lib/matplotlib/dates.py

```diff
@@ -99,6 +99,8 @@
 
 .. _dateutil: https://dateutil.readthedocs.io
 
+.. _date-tickers:
+
 Date tickers
```

Comment:
This should also probably be changed to `Date tick locators`?

Reply:
Yes, I just named the reference after the heading, if the heading changes it would make sense to change the reference along (although technically not necessary).